astropy/modeling/rotations.py

```diff
@@ -2,7 +2,7 @@
 
 """
 Implements rotations, including spherical rotations as defined in WCS Paper II
-[1]_
+[1]_.
```

Comment:
Move reference to the sentence below? (and full stop after `II`)

Reply:
✅ It actually fits well on 1 line.